### ISSUE TITLE
Defect 2a - Critical - Content Strategy Statement Image

### ISSUE THREAD

**Issue by srancour** [department-of-veterans-affairs/va.gov-cms#19371]
## Description or Additional Context
The Content Strategy Statement is an image with lots of detailed text but a limited alt text of "Diagram of the CMS Knowledge Base Content Principles Statement".

## Example
The image used on [CMS Knowledge Base content strategy statement](https://prod.cms.va.gov/help/cms-knowledge-base-content-strategy-statement) has limited alt text for how much text is in it.

### Code snippet
Image `<img loading="lazy" src="/sites/default/files/styles/large/public/2021-07/CMS%20knowledge%20base%20content%20strategy%20principles.jpg" width="480" height="315" alt="Diagram of the cms knowledge base content strategy principles" typeof="foaf:Image" class="image-style-large" naptha_cursor="text">`

### Steps to reproduce
Navigate: Home -> Knowledge Base -> How to find Content to Edit -> CMS Knowledge Base content strategy principle
Image is under first heading.

## Recommended resolution
Since all of the text in the image is below the image in plain text, the alt text of this image is a non-issue.

All images in the KB should be checked to make sure that the alt text makes sense.

## Defect Criterion
[WCAG 1.1.1 - Non-text Content](https://www.w3.org/TR/WCAG21/#non-text-content) - All [non-text content](https://www.w3.org/TR/WCAG21/#dfn-non-text-content) that is presented to the user has a [text alternative](https://www.w3.org/TR/WCAG21/#dfn-text-alternative) that serves the equivalent purpose, except for the situations listed below.

### Users affected
- Screen reader user

## Teams that this affects:
- [x] CMS Team
- [x] sitewide

**Comment by srancour** [comment]
Fixed alt-text for image
